client/conf/java.security.ciphers.in

```diff
@@ -15,4 +15,21 @@
 # specific language governing permissions and limitations
 # under the License.
 
-jdk.tls.disabledAlgorithms=SSLv2Hello, SSLv3, DH keySize < 128, RSA keySize < 128, DES keySize < 128, SHA1 keySize < 128, MD5 keySize < 128, RC4
+jdk.tls.disabledAlgorithms=SSLv3,TLSv1.0, TLSv1.1, RC4, DES, MD5withRSA, DH keySize < 2048, EC keySize < 224, \
```

Comment:
chances are this could break support of several older/supported hypervisors, can you share what you've tested with these changes @fermosan ?

Reply:
I tested it only against KVM (Ubuntu 20.04). I could try to install one or two hypervisors and test it but what is the point? All supported hypervisors must be tested to make sure the change is ok.

Is it possible to use different definitions for type of service ? 
For example have one definition for management server login portal and one for all the other backend communications. The big issue is the outside connections from users to the UI or/and apis and not the hypervisor communication.